smallrye-reactive-messaging-amqp/src/test/java/io/smallrye/reactive/messaging/amqp/AmqpFailureHandlerTest.java

```diff
@@ -131,6 +140,46 @@ public void testRejectStrategy() {
         assertThat(isAmqpConnectorReady(connector)).isTrue();
     }
 
+    @Test
+    public void testModifiedFailedStrategy() {
+        getModifiedFailedConfig();
+        MyReceiverBeanRecovering bean = deployRecovering();
+        AtomicInteger counter = new AtomicInteger();
+        AmqpConnector connector = container.getBeanManager().createInstance().select(AmqpConnector.class,
+                ConnectorLiteral.of(AmqpConnector.CONNECTOR_NAME)).get();
+        await().until(() -> isAmqpConnectorReady(connector));
+        await().until(() -> isAmqpConnectorAlive(connector));
+
+        usage.produceTenIntegers("modified-failed", counter::getAndIncrement);
+
+        await().atMost(2, TimeUnit.MINUTES).until(() -> bean.list().size() >= 10);
+        // All messages should not have been received + 3, 6 and 9 are redelivered
```

Comment:
Should have been received?

Reply:
yes, we capture the list before the failure.

Reply:
I mean that the comment says "All messages should **not** have been received", while the test expects they were all recieved (plus a few twice), and wont get that far unless enough were. It then goes on to assert the exact paylods, so it seems like the comment should be  "All messages should have been received". Or maybe "now" instead of "not" might work too.

Same for other instance below